image/scanner/konflux/Dockerfile

```diff
@@ -0,0 +1,77 @@
+ARG BASE_REGISTRY=registry.access.redhat.com
```

Comment:
[about the file path]
I realized there's now inconsistency. This file is
`image/scanner/konflux/Dockerfile`
whereas `scanner-db`-s is
`image/db/rhel/konflux.Dockerfile`

I suggest moving this file to `image/scanner/rhel/konflux.Dockerfile` for consistency. WDYT?

Reply:
I agree.